packages/tasks/src/pipelines.ts

```diff
@@ -693,6 +693,12 @@ export const PIPELINE_DATA = {
 		color: "yellow",
 		hideInDatasets: true,
 	},
+	"molecules": {
+		name: "Molecules",
+		modality: "other",
+		color: "green",
+		hideInDatasets: true,
+	},
```

Comment:
maybe it's more for domains where we have biology etc?

Reply:
i.e. in there (internal codebase):

```ts
export const OTHER_TAGS_SUGGESTIONS = [
	"chemistry",
	"biology",
	"finance",
	"legal",
	"music",
	"art",
	"code",
	"climate",
	"medical",
	TAG_NFAA_CONTENT,
] as const;
```

Reply:
+1 and doesn't Chemistry cover molecules? - a bit wary of adding too niche tags

Reply:
Where are those domains meant to be displayed? I would prefer just a "Biology" tag next to the current "Graph machine learning" one to cover that domain (including protein folding, DNA sequence models, molecules and more).

Reply:
I think it's a different output type with enough number of models (although represented by text), a bit like 3D in a way.

Reply:
this model for instance outputs structured output